skorecard/bucket_mapping.py

```diff
@@ -66,8 +66,8 @@ def __post_init__(self) -> None:
         assert self.type in ["numerical", "categorical"]
         assert len(self.map) is not None, "Please set a 'map' first"
         assert isinstance(self.specials, dict) or isinstance(self.specials, list)
-        if self.missing_bucket is not None:
-            assert isinstance(self.missing_bucket, int)
+        # if self.missing_bucket is not None:
```

Comment:
Remove commented lines

Reply:
done for `passthrough` method, will add an issue to continue this theme